market/utils/automated-process-start-utils/meta.json

```diff
@@ -0,0 +1,21 @@
+{
+	"id": "automated-process-start-utils",
+	"name": "Automated Process Start Utils",
+	"description": "Automated Process Start Utils is a job-firing schedule that recurs based on calendar-like notions.",
+	"type": "solution",
+	"platformReview": "4.5",
+	"sourceUrl": "https://github.com/axonivy-market/automated-process-start-utils",
+	"statusBadgeUrl": "https://github.com/axonivy-market/automated-process-start-utils/actions/workflows/ci.yml/badge.svg",
+	"language": "English",
+	"industry": "Cross-Industry",
+	"tags": ["utils"],
+	"validate": true,
+	"mavenArtifacts": [
+		{
+			"name": "Automated Process Start Utils Product",
+			"groupId": "com.axonivy.utils.automatedprocessstartutils",
```

Comment:
Not happy about this long name and also about the name of the market artifact. 

what is difference to the component in portal?

Reply:
Hi @alexsuter 

it is the same but my task was to isolate it from the portal and make it as connector in the marketplace.

Reply:
great! we should find a more expressive and shorter name.

Reply:
What about "Cron Trigger Process"?

Reply:
@andreasbalsiger do you have good names for this new market product?

Reply:
I've talked to Andreas. Why not using cron jobs?

Reply:
we better should use a clock 🕐 as logo and not quartz. because as far as I know, the interface is independent from the technology?

Reply:
> I've talked to Andreas. Why not using cron jobs?

so only "Cron Job" instead of "Automated Process Start Utils"?